### DISCUSSION TITLE
What are some important journal articles should to read if you want study ML/AI?

### DISCUSSION THREAD

**Discussion by AnzhiZhang** [Schweinepriester/github-profile-achievements#674] [Q&A] [answered]
Any information or tips?

**Comment by ArielMAJ** [comment, marked as answer]
The most important journal on Computer Vision right now is [IEEE/CVF Conference on Computer Vision and Pattern Recognition (CVPR)](https://ieeexplore.ieee.org/xpl/conhome/1000147/all-proceedings). It's the 4th biggest and most important journal in the world. You can find the best articles and most relevant/recent information on the area there.
